samples/openapi3/client/petstore/dart-dio/petstore_client_lib_fake/lib/model/map_test.dart

```diff
@@ -30,18 +30,18 @@ abstract class MapTest implements Built<MapTest, MapTestBuilder> {
     static Serializer<MapTest> get serializer => _$mapTestSerializer;
 }
 
-class MapTestInnerEnum extends EnumClass {
+class MapTestMapOfEnumStringEnum extends EnumClass {
 
   @BuiltValueEnumConst(wireName: r'UPPER')
-  static const MapTestInnerEnum UPPER = _$mapTestInnerEnum_UPPER;
+  static const MapTestMapOfEnumStringEnum UPPER = _$mapTestMapOfEnumStringEnum_UPPER;
   @BuiltValueEnumConst(wireName: r'lower')
-  static const MapTestInnerEnum lower = _$mapTestInnerEnum_lower;
+  static const MapTestMapOfEnumStringEnum lower = _$mapTestMapOfEnumStringEnum_lower;
 
-  static Serializer<MapTestInnerEnum> get serializer => _$mapTestInnerEnumSerializer;
+  static Serializer<MapTestMapOfEnumStringEnum> get serializer => _$mapTestMapOfEnumStringEnumSerializer;
```

Comment:
`mapTestMapOfEnumStringEnumSerializer` :laughing:

Reply:
 😄 Well to be fair, its is private and generated to prevent collisions